docs/reference/indices/flush.asciidoc

```diff
@@ -93,7 +93,7 @@ which returns something similar to:
                {
                  "commit" : {
                    "id" : "3M3zkw2GHMo2Y4h4/KFKCg==",
-                   "generation" : 3,
+                   "generation" : 4,
```

Comment:
hmm why did this change?

Reply:
To create an empty shard, we now:
1) Create an empty index via `Store.createEmpty`
2) Create an empty translog
3) Associate it with the index using `Store. associateIndexWithNewTranslog` 

This creates one more commit compare to how it was to be. I can change createEmpty to require a translogUUID as a parameter , if you prefer. I'm OK either way.